students/5/main.js

```diff
@@ -106,8 +108,22 @@ const getNamesShortZip = async () => {
   return getNames;
 };
 
+//exercise 6
+const getPostWithName = async (postId) => {
+  let postName = [];
+  try {
+    userId =1;
```

Comment:
We can't really do this (`userId = 1`).  This is an example of hard coding and doesn't make our function work with other userIds.